### DISCUSSION TITLE
For inpaint speed?

### DISCUSSION THREAD

**Discussion by peki12345** [lllyasviel/Fooocus#3256] [Q&A] [answered]
I know that inpainting is a combination of text2img and lora. I want to improve the speed of inpainting, so I replaced the text2img model with a 4-step model from Hyper-SD and used 4-step inference. However, I found that there is a lot of noise in the results. Is this method incorrect? How can I improve the speed of inpainting?

**Comment by mashb1t** [comment, marked as answer]
You can try to reduce the noise by setting the inpaint engine to None instead of v2.6 to not encounter issues with e.g. HyperSD.
The inpaint model has been trained on SDXL and works best with 30 steps.
